docker/checkout.bash

```diff
@@ -102,7 +102,7 @@ cd ${dir}/src
 
 # git clone --branch master --depth 1 https://github.com/nitroshare/qhttpengine.git
 git clone -b master --depth 1 https://github.com/etherealjoy/qhttpengine.git
```

Comment:
We should be pulling a fixed version for any dependencies. Can someone log this as an issue. We also need a issue to remove carma-cloud service/client into separate library so both V2X-Hub and Platform can consume it.